src/plugins/Transloadit/index.js

```diff
@@ -65,6 +66,10 @@ module.exports = class Transloadit extends Plugin {
       })
 
       this.core.setState({ files })
+
+      if (this.opts.wait) {
+        return this.beginWaiting()
+      }
     }).catch((err) => {
       this.core.emit('informer', '⚠️ Transloadit: Could not create assembly', 'error', 0)
```

Comment:
Would be cool if the error was accessible somehow. Maybe behind a `more details` link for the user. But i realize that goes beyond the scope of this PR, so maybe we can just dump it to the `console.error` for now short of having sth like that?